src/connectors/numeric-refinement-list/connectNumericRefinementList.js

```diff
@@ -33,8 +35,7 @@ Full documentation available at https://community.algolia.com/instantsearch.js/v
 /**
  * @typedef {Object} NumericRefinementListItem
  * @property {string} label Name of the option.
- * @property {number} start Lower bound of the option (>=).
- * @property {number} end Higher bound of the option (<=).
+ * @property {string} value URL encoded value of the form `min:max`
```

Comment:
Actually the output is a bit more ugly as it is a JSON object with start and end keys that is URL encoded (with the keys). The idea when we choose this direction was to consider the value as a black box (even though it's really poorly documented). Proposal:

> URL encoded of the bounds object with the form `{start, end}`. This value can be used verbatim in the webpage and can be read by `refine` directly. If you want to inspect the value, you can do `JSON.parse(window.decodeURI(value))` to get the object.